src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -21,43 +30,83 @@ export default createRule<Options, MessageIds>({
     version: '>=12.0.0',
     docs: {
       category: 'Possible Errors',
-      description:
-        'Use `concatLatestFrom` instead of `withLatestFrom` to prevent the selector from firing until the correct `Action` is dispatched.',
+      description: `Use \`${concatLatestFromKeyword}\` instead of \`${withLatestFromKeyword}\` to prevent the selector from firing until the correct \`Action\` is dispatched.`,
       recommended: 'warn',
-      suggestion: true,
     },
-    schema: [],
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          strict: {
+            type: 'boolean',
+            default: defaultOptions.strict,
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
     messages: {
-      [messageId]: 'Use `concatLatestFrom` instead of `withLatestFrom`.',
-      [messageIdSuggest]: 'Replace `withLatestFrom` with `concatLatestFrom`.',
+      [messageId]: `Use \`${concatLatestFromKeyword}\` instead of \`${withLatestFromKeyword}\`.`,
     },
   },
-  defaultOptions: [],
-  create: (context) => {
+  defaultOptions: [defaultOptions],
+  create: (context, [{ strict }]) => {
+    const actionsName = findNgRxEffectActionsName(context)
+    if (!actionsName) return {}
+
+    const sourceCode = context.getSourceCode()
+    const selector = strict
+      ? (`${createEffectExpression} CallExpression > Identifier[name='withLatestFrom']` as const)
+      : (`${createEffectExpression} ${storeExpression(
```

Comment:
Ehh, maybe we should rename this selector in another PR. Actually `storeExpression` represents something named `XYZ` that is injected in `constructor`, not necessarily `store: Store` or `<accessibility> <readonly> store: Store`.